esp/esp/django_settings.py

```diff
@@ -51,6 +51,7 @@
 # Must be unique for every site hosted
 CACHE_PREFIX="ESP"
 
+ALLOWED_HOSTS = ['*']
```

Comment:
I was about to say "this isn't a regression, let's fix it some other time" but then I noticed how easy to fix it is.